src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -48,16 +52,32 @@ private static string[] GetBuiltinExtensionAssemblies()
 
         public Type[] GetStartupTypes()
         {
-            IEnumerable<Type> startupTypes = GetExtensionsStartupTypes();
+            IEnumerable<Type> startupTypes = GetExtensionsStartupTypesAsync().ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Changed the mehtod to async but making this a sync call? If possible we should avoid this pattern

Reply:
It is unavoidable at this place since this is expected by webjobs sdk.